package.json

```diff
@@ -84,7 +84,6 @@
     "focus-visible": "^5.2.0",
     "glob": "^7.1.6",
     "lodash.isempty": "^4.4.0",
```

Comment:
It should be safe to remove this! It's not explicitly imported by anything. If it's needed by another dependency it'll still get included in the lock file. It's not hurting our bundle for it to be here, but someone might accidentally use it :stuck_out_tongue: